src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4008,8 +4008,10 @@ private BoundExpression MakeBadExpressionForObjectCreation(ObjectCreationExpress
                 expanded,
                 argsToParamsOpt,
                 resultKind,
-                boundMember.Type,
-                hasErrors);
+                binderOpt: this,
+                useSetterForDefaultArgumentGeneration: true,
```

Comment:
I'm suggesting `useSetterForDefaultArgumentGeneration` should not exist if it is always `true`. I could not find code that could set it to `false`, but we should double-check.